config.rb

```diff
@@ -143,6 +144,7 @@ def self.test?
   override :postgres_service_hostname, "postgres.ubicloud.com", string
   override :postgres_monitor_database_url, Config.clover_database_url, string
   optional :postgres_monitor_database_root_certs, string
+  override :postgres_monitor_database_rds_iam_auth_enabled, false, bool
```

Comment:
It's technically possible for the monitor database to be different and require different auth from the main database, but is that a situation you actually need to support? If not, probably simpler to only have a single option, and let it control both.